src/domain/metric/service/postMetric.js

```diff
@@ -30,7 +31,7 @@ export default (posts: Post[]): PostMetric[] => {
     return {
       post,
       id: post.id,
-      createdTime: new Date(post.created_time),
+      createdTime: moment(post.created_time).format(),
```

Comment:
@hellowin please refer to this http://stackoverflow.com/questions/4310953/invalid-date-in-safari